pkg/catalogv2/helmop/operation.go

```diff
@@ -1149,6 +1149,12 @@ func (s *Operations) AddCpTaintsToTolerations(tolerations []corev1.Toleration) (
 	if err != nil {
 		return nil, fmt.Errorf("failed to list control plane nodes: %w", err)
 	}
+	if len(cpList.Items) == 0 {
```

Comment:
We can't know what label the node has, as it depends on which k8s is installed, i.e rke1 or rke2. We only do it in case cpList is empty because the nodes won't have both labels